.github/workflows/update-upstream-provider.yml

```diff
@@ -8,7 +8,7 @@ env:
   PUBLISH_REPO_USERNAME: ${{ secrets.OSSRH_USERNAME }}
   PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
   PULUMI_API: https://api.pulumi-staging.io
-  PULUMI_EXTRA_MAPPING_ERROR: true
+  PULUMI_EXTRA_MAPPING_ERROR: true  
```

Comment:
Spurious diff?

Reply:
this will get fixed this week when we roll out the new GHA for the Java SDK